configure.ac

```
@@ -3677,7 +3680,7 @@ AS_IF([test "$EXEEXT" = .exe], [
 
 AS_CASE("$cross_compiling:${LIBPATHENV}", [yes:* | no:], [], [
     AC_MSG_CHECKING(whether wrapper for $LIBPATHENV is needed)
-    AS_IF([env ${LIBPATHENV}=/lib /bin/sh -c ': ${'${LIBPATHENV}'?}' 2>/dev/null],
+    AS_IF([test x"$enable_shared" = xno],
```

We may need to consider WCHAR env on Windows.

```suggestion
    AS_CASE(["$enable_shared:$target_os"], [no:*|*:mingw*|*:cygwin],
```